plugin.rb

```diff
@@ -20,6 +20,7 @@ class Engine < ::Rails::Engine
 
     def self.clean_tag(tag)
       tag.downcase.strip[0...SiteSetting.max_tag_length].gsub(TAGS_FILTER_REGEXP, '')
+                                                        .gsub(/\.(json|rss)$/, '')
```

Comment:
I am not certain this is the correct approach. Instead, shouldn't it not append the mime type when making requests to these actions?